internal/httpcore/presenter.html

```diff
@@ -101,39 +101,40 @@ <h3>or go to <a href="{{url}}">{{url}}</a></h3>
         // const messageLine = document.getElementById("messageline")
         const currentAction = document.getElementById("action")
         const voteCodeDisplay = document.getElementById("vote-code")
-        const nowElectingDisplay = document.getElementById("now-electing")
-        const nowElectingRole = document.getElementById("now-electing-role")
+        const nowVotingDisplay = document.getElementById("now-voting")
+        const nowVotingThing = document.getElementById("now-voting-thing")
 
         const electionStart = (data) => {
             console.log("election start")
-            nowElectingRole.innerText = data.roleName
+            nowVotingThing.innerText = data.roleName ?? data.title
```

Comment:
`Uncaught TypeError: can't access property "roleName", data is undefined` when opening voting page while a poll of any type is open for voting.